src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj

```diff
@@ -26,7 +26,9 @@
     <Compile Include="System\Resources\Extensions\TypeNameComparer.cs" />
     <Compile Include="$(CoreLibSharedDir)System\Numerics\Hashing\HashHelpers.cs"
              Link="System\Numerics\Hashing\HashHelpers.cs" />
+    <Compile Include="$(CoreLibSharedDir)System\Diagnostics\CodeAnalysis\RequiresUnreferencedCodeAttribute.cs" />
```

Comment:
Add Link attribute?

Reply:
`Link=""` just impacts where it shows up in VS in the solution explorer

Reply:
It is needed because this OOB assembly targets netstandard2.0 where the attribute is not present, hence we compile it (without exposing it) which is what we have done in other similar situations as part of this effort.

---
In reply to: [569249684](https://github.com/dotnet/runtime/pull/47778#discussion_r569249684) [](ancestors = 569249684)